dynamic-demo-plugin/OWNERS

```diff
@@ -0,0 +1,23 @@
+reviewers:
+  - dtaylor113
+  - jcaianirh
+  - jeff-phillips-18
+  - jhadvig
+  - kyoto
+  - rhamilto
+  - spadgett
+  - TheRealJon
+  - zherman0
```

Comment:
Maybe @vojtechszocs @rawagner @spadgett @christianvogt @andrewballantyne @glekner for reviewers.